src/components/example-GuitarRecommendation.tsx

```
@@ -5,43 +5,43 @@ import { showAIAssistant } from "../store/example-assistant";
+            ${guitar.price}
+          </div>
+          <button
+            onClick={() => {
+              navigate({
+                to: "/example/guitars/$guitarId",
+                params: { guitarId: guitar.id.toString() },
+              });
+              showAIAssistant.setState(() => false);
+            }}
+            className="bg-gradient-to-r from-orange-500 to-red-600 text-white px-4 py-1.5 rounded-lg text-sm hover:opacity-90 transition-opacity"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)